tools/cli.js

```diff
@@ -109,17 +109,18 @@ const state = {
   connection: null
 };
 
-commandProcessor._complete = (inputs, depth) => {
+commandProcessor.complete = (inputs, depth) => {
+  const completions = ['call', 'event', 'connect', 'disconnect', 'exit'];
```

Comment:
I'd prefer these to be sorted alphabetically, though I don't feel strongly for it.